components/infoboxes/commons/infobox_series.lua

```diff
@@ -7,6 +7,7 @@ local Namespace = require('Module:Namespace')
 local Locale = require('Module:Locale')
 local ReferenceCleaner = require('Module:ReferenceCleaner')
 local Links = require('Module:Links')
+local Flags = require('Module:Flags')._Flag
```

Comment:
Would Module:Flags/MasterData be better?

Reply:
Module:Flags uses Module:Flags/MasterData (first line: `local FlagData = mw.loadData('Module:Flags/MasterData')`)
it standardizes the input, has some alias support, supports 2&3 letter codes and adds a fallback to the safeExpand if it can not find anything in the MasterData

Reply:
Alright